src/state/CartProvider.jsx

```diff
@@ -1,103 +1,51 @@
-import React, { useReducer, useContext } from 'react'
+import React, { createContext, useReducer, useContext } from 'react';
 
-// Initialize the context
-const CartContext = React.createContext()
-
-// Definte the default state
+// Initial state of the cart
 const initialState = {
-  itemsById: {},
-  allItems: [],
-}
-
-// Define reducer actions
-const ADD_ITEM = 'ADD_ITEM'
-const REMOVE_ITEM = 'REMOVE_ITEM'
-const UPDATE_ITEM_QUANTITY = 'UPDATE_ITEM_QUANTITY'
+  cartItems: []
```

Comment:
For this assignment, the state needs to stay normalized with the `allItems` and `itemsById`